src/main/java/makaroshyna/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package makaroshyna.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
```

Comment:
do not forget about quals, hashcode & toString, but exclude lazy-initialized fields from it